### PR TITLE
update Quick Start

### PR DESCRIPTION
**Pull request by rosemarymarano**
Add information for accessing the WLS Administration Console.

### CODE REVIEW

File: site/quickstart.md [hunk 1]
```diff
@@ -249,6 +249,11 @@ $ curl -v -H 'host: sample-domain1.org' http://your.server.com:30305/weblogic/
 **Note**: Depending on where your Kubernetes cluster is running, you may need to open firewall ports or
 update security lists to allow ingress to this port.
 
+f.	To access the WLS Administration Console, uncomment the `NodePort` setting in your `domain.yaml` file and open a browser to http://your.server.com:30305. Substitute the Node IP address of the worker node for `your.server.com`. You can find it by running:
```
Comment (mriccell) [inline site/quickstart.md]: in the Quick start guide sample the domain.yaml is generated by entries in the template  `kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/create-domain-inputs.yaml`.  There are instructions before to copy the sample `kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/create-domain-inputs.yaml` and a note For example, assuming you named your copy `my-inputs.yaml`.

Comment (mriccell) [reply site/quickstart.md]: I would say 
To access the WLS Administration Console, uncomment the `NodePort` setting in my-inputs.yaml (assuming you named your copy my-inputs.yaml) file and open a browser to http://your.server.com:30305.

Comment (rosemarymarano) [reply site/quickstart.md]: Please review these changes to make sure that I got in the information you wanted. Thanks.

File: site/quickstart.md [hunk 2]
```diff
@@ -246,8 +252,10 @@ $ curl -v -H 'host: sample-domain1.org' http://your.server.com:30305/weblogic/
  < Vary: Accept-Encoding
 < * Connection #0 to host your.server.com left intact
 ```
-**Note**: Depending on where your Kubernetes cluster is running, you may need to open firewall ports or
-update security lists to allow ingress to this port.
+
+**Note**: Depending on where your Kubernetes cluster is running, you may need to open firewall ports or update security lists to allow ingress to this port.
+
+f.	To access the WLS Administration Console, edit the `my-inputs.yaml` file (assuming that you named your copy `my-inputs.yaml`) to set `exposedAdminNodePort: true`. Open a browser to `http://your.server.com:30305`. As in the previous step, substitute the Node IP address of the worker node for `your.server.com`.
```
Comment (mriccell) [inline site/quickstart.md]: The port is 30701 not 30305

Comment (rosemarymarano) [reply site/quickstart.md]: Changed all references from 30305 to 30701.

**Pull request merged**